addons/nuxeo-retention/nuxeo-retention-web-ui/src/main/resources/web/nuxeo.war/ui/document/retentionrule/nuxeo-retentionrule-view-layout.html

```diff
@@ -69,7 +69,7 @@
         <div role="widget">
           <label>[[i18n("retention.rule.label.startPolicy.eventBased.expression.placeholder")]]</label>
           <nuxeo-tag name="eventInput">
-            [[document.properties.retention_def:startingPointExpression]]
```

Comment:
Does this pass linting? If yes, it is ok for me!

Reply:
I think yes, I compared it to the others ones, it looks the same formatting :)